liblangutil/CharStream.cpp

```diff
@@ -73,6 +73,12 @@ char CharStream::rollback(size_t _amount)
 	return get();
 }
 
+char CharStream::seek(size_t _location)
```

Comment:
Ah also `seek` at least as a unix function can have different modes, so it is not clear whether this is a relative or an absolute setting. Perhaps `setPosition`?